### ISSUE TITLE
New Pull Request template refers to a `dev` branch, which does not exist

### ISSUE THREAD

**Issue by yifanmai** [stanford-crfm/mistral#185]
The New Pull Request template says:

```
**BEFORE YOU START**: please make sure your pull request is against the `dev` branch.
We cannot accept pull requests against the `main` branch.
```

We should either change this, or create the `dev` branch.

**Comment by dlwh** [comment]
Yeah, the dev branch lived for a long long time and we deleted it not that long ago. I sort of prefer doing work off of main, but open to going back to the way it was. @J38 @yifanmai what do y'all think?

**Comment by yifanmai** [comment]
I prefer working off `main` and avoiding the extra work of periodically syncing the branches.
